guides/common/modules/proc_adding-custom-deb-repository-example-for-debian-11.adoc

```diff
@@ -0,0 +1,42 @@
+[id="Adding_Custom_DEB_Repository_Example_for_Debian_11"]
+= Adding {customdebtitle} Repository Example for Debian 11
+
+This example creates a product and repositories for Debian 11.
+
+.Prerequisites
+. xref:Extracting_GPG_Public_Key_Fingerprints_from_a_Release_Files_{context}[Extract the GPG public keys] from the `Release` file
+. xref:Importing_a_Custom_GPG_Key_{context}[Import the content credentials] into {Project}
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content > Products*.
+. Click *Create Product* to xref:Creating_a_Custom_Product[create a product] named `Debian 11`.
+. On the *Repositories* tab, click *New Repository* to create three repositories of type `deb` as follows:
++
+* *Debian 11 main*
+** *URL*: `\http://ftp.debian.org/debian/`
```

Comment:
Out of my curiousity, whats the `\` escape for? To avoid rendering it as a link? I would expect asciidoctor not render a verbose text as links.

Reply:
Yes, to avoid rendering the link. In mostly unnecessary when you have something like "foreman.example.com/some/path" which is never reachable. In this specific case, you really don't need to browser debian.org at all.